code/modules/mob/living/simple_animal/hostile/megafauna/stalwart.dm

```diff
@@ -0,0 +1,176 @@
+/mob/living/simple_animal/hostile/megafauna/stalwart
+	name = "stalwart"
+	desc = "A graceful, floating automaton. It emits a soft hum."
+	health = "3000"
```

Comment:
health is a number not a string, same for max_health

Reply:
was I high when I read this?